src/FSharp.Core/prim-types.fs

```diff
@@ -4111,6 +4134,19 @@ namespace Microsoft.FSharp.Collections
         
     and 'T list = List<'T>
 
+#if NETSTANDARD2_1_OR_GREATER
+    and [<CompilerMessage("This type is for compiler use and should not be used directly", 1204, IsHidden=true);
+          Sealed;
+          AbstractClass;
+          CompiledName("FSharpList")>] List =
+        [<CompilerMessage("This method is for compiler use and should not be used directly", 1204, IsHidden=true)>]
+        static member Create([<System.Runtime.CompilerServices.ScopedRef>] items: System.ReadOnlySpan<'T>) =
+            let mutable list : 'T list = []
+            for i = items.Length - 1 downto 0 do
+                list <- items[i] :: list
+            list
```

Comment:
Curious if it's worth it to use a listcollector here, and only then convert to list.

Reply:
`ListCollector<'T>` doesn't exist yet here.

I think this should be at least as good, though, anyway, right?

This is how `ofArray` is implemented as well:

https://github.com/dotnet/fsharp/blob/00f025c7eaaffb1e00c6e0c13a2467e4d79ca97f/src/FSharp.Core/local.fs#L529-L533

Reply:
Ah, I see. Probably should be good, yes.